backend/src/v1/routes/userRoutes.js

```diff
@@ -5,7 +5,9 @@ const userController = require('../controllers/userController');
 
 const router = express.Router();
 
-router.get('/', userController.getAllUsers);
+router.get('/', bodyParser.json(), userController.getAllUsers);
+
+router.get('/user?', bodyParser.json(), userController.getUser);
```

Comment:
Looking at this now, I think it looks cleaner using a route parameter.

change `router.get('/user?')` to `router.get('/user/:username')`